guillotina/tests/test_postgres.py

```diff
@@ -329,34 +331,56 @@ async def test_should_not_resolve_conflict_error_with_resolve(db, dummy_guilloti
         await cleanup(aps)
 
 
-@pytest.mark.skipif(DATABASE in ("cockroachdb", "DUMMY"), reason="Cockroach not support simple...")
-async def test_should_not_resolve_conflict_error_with_simple_strat(db, dummy_guillotina):
```

Comment:
I removed this test because with this change ins't true anymore. Conflicts only happen when two transactions modify same object